### DISCUSSION TITLE
Question:  Differences between the 2 Appimages?

### DISCUSSION THREAD

**Discussion by est61** [srevinsaju/guiscrcpy#341] [General]
Thanks for this amazing project!

Can you explain the differences between the 2 Appimages?
Also would be nice to have this in the FAQ

**Comment by srevinsaju** [comment]
Sure, 
There are two appimages
`guiscrcpy*.r.AppImage` is the complete installation of guiscrcpy, which includes, `scrcpy`, `adb` and `guiscrcpy` (scrcpy is the command line tool to display images, `guiscrcpy` only caters to running it using the gui

`guiscrcpy-min*.AppImage` are a minimal installation of `guiscrcpy`. It DOES NOT include `scrcpy` and `adb`. You will have to manually install them.

Now, `guiscrcpy-min*.AppImage` will run even on very old Linux'es, i.e those released even before 2016. But the full version will only run on Linuxes released after 2018. This is indicated with the difference in the `glibc` version in the release assets.

To see, if your computer is compatible, run
```bash
ldd --version
```

This will give something like this
```bash
❯ ldd --version
ldd (GNU libc) 2.32
Copyright (C) 2020 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
Written by Roland McGrath and Ulrich Drepper.
```

You can see, mine is 2.32. That means, I can run both of the appimages, but consider the version is displayed `2.20`. That means you cannot run the full version, because that requires at least `2.23` otherwise it will crash with a `GLIBC_X_X` not found error.

Let me know if you need any more clarification, I happy to explain

**Comment by crass** [comment]
I agree, this would be nice to add to the FAQ or in the release notes. I was looking for answers to the same question and just found this issue.

**Comment by askusWT** [comment]
Agree, this would be so helpful to find on the releases or by the assets!
